src/pkg/utils/utils.go

```diff
@@ -643,13 +643,10 @@ func ShortID(id string) string {
 // 'general.distro' key in a config file or 'fedora') is assumed.
 func ParseRelease(distro, release string) (string, error) {
 	if distro == "" {
-		distro = distroDefault
-		if viper.IsSet("general.distro") {
-			distro = viper.GetString("general.distro")
-		}
+		distro, _ = ResolveDistro(distro)
```

Comment:
To continue my [comment](https://github.com/containers/toolbox/pull/977#discussion_r960631867) from the previous commit ...

The `ResolveDistro` call is not just repeated across the front-end as part of the overall resolution code.  It's now spreading across the backend, and, since the function is fallible it makes me wonder if different invocations of it can yield different results.  It will be better if we could just resolve the `distro` only once and keep re-using the value.

Reply:
Container and image name resolution is re-unified in https://github.com/containers/toolbox/pull/1080